src/components/TextInput/styles.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+
+export const InpuText = styled.input`
+  width: 300px;
+  height: 35px;
+  outline: none;
+  margin: 0;
+  margin-left: 5px;
+  padding: 6px;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  background: #ffffff;
+  border: 1px solid #808080;
+  border-radius: 8px;
+`;
+export const LabelText = styled.label`
+  color: #808080;
+  font-family: Open Sans;
```

Comment:
The setup of the font is not needed here as it has been included in the html tag, but you must include it in the input properties because the html property does not affect the form fields.